### PR TITLE
Edit AboutUs.md

### PR DESCRIPTION
**Pull request by ThisisXXZ**
Tutorial Week 7 - conflict resolve practices

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -2,8 +2,4 @@
 
 Display | Name | Github Profile | Portfolio 
 --------|:----:|:--------------:|:---------:
-![](https://via.placeholder.com/100.png?text=Photo) | John Doe | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
-![](https://via.placeholder.com/100.png?text=Photo) | Don Joe | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
-![](https://via.placeholder.com/100.png?text=Photo) | Ron John | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
-![](https://via.placeholder.com/100.png?text=Photo) | John Roe | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
-![](https://via.placeholder.com/100.png?text=Photo) | Don Roe | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
+![](https://via.placeholder.com/100.png?text=Photo) | Xiong Xinzhuang | [Github](https://github.com/ThisisXXZ) | [Portfolio](docs/team/thisisxxz.md)
```
Comment (ehz0ah) [inline docs/AboutUs.md:5]: Looks good!

**Review by ehz0ah — APPROVED** [review]
LGTM!

File: docs/AboutUs.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 # About us
 
-Display | Name | Github Profile | Portfolio 
---------|:----:|:--------------:|:---------:
-![](https://via.placeholder.com/100.png?text=Photo) | Xiong Xinzhuang | [Github](https://github.com/ThisisXXZ) | [Portfolio](docs/team/thisisxxz.md)
\ No newline at end of file
+Display |    Name     |           Github Profile            | Portfolio 
+--------|:-----------:|:-----------------------------------:|:---------:
+![](https://picsum.photos/seed/picsum/200/300) | Lee Hao Zhe | [Github](https://github.com/ehz0ah)
```
Comment (ehz0ah) [inline docs/AboutUs.md:5]: Good job merging conflicts!

**Review by ehz0ah — DISMISSED** [review]
Well done!

File: docs/AboutUs.md [hunk 3]
```diff
@@ -1,9 +1,6 @@
...
-Display | Name | Github Profile | Portfolio 
---------|:----:|:--------------:|:---------:
-![](https://via.placeholder.com/100.png?text=Photo) | John Doe | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
-![](https://via.placeholder.com/100.png?text=Photo) | Don Joe | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
-![](https://via.placeholder.com/100.png?text=Photo) | Ron John | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
-![](https://via.placeholder.com/100.png?text=Photo) | John Roe | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
-![](https://via.placeholder.com/100.png?text=Photo) | Don Roe | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
+Display |    Name     |           Github Profile            | Portfolio 
+--------|:-----------:|:-----------------------------------:|:---------:
+![](https://picsum.photos/seed/picsum/200/300) | Lee Hao Zhe | [Github](https://github.com/ehz0ah)
```
Comment (ehz0ah) [inline docs/AboutUs.md:5]: GOOD JOB!

**Review by ehz0ah — APPROVED** [review]
Time to merge!

File: docs/AboutUs.md (hunk 3 above)
Comment (isaacsaw25) [inline docs/AboutUs.md:5]: Outstanding work brother

**Review by isaacsaw25 — DISMISSED** [review]
LGTM!

**Review by paulktham — DISMISSED** [review]
LGTM!

**Comment by isaacsaw25** [comment]
help

**Pull request closed without merging**